src/utils/types.ts

```diff
@@ -8,7 +8,8 @@ export interface StaticValue {
 
 export interface ConditionalValue {
   test: string;
-  then: any;
+  // then is reserved for thenables
+  ifTrue: any;
   else: any;
 }
```

Comment:
Went with a new (and I think better) solution!

`then` is reservered for thenables, so changed everything in the code from `then` -> `ifTrue`.